crane-cloud/backend
# Backend resource cleanup

# Description

This cleanup involves removing organisations, organisation_members and organisation_admins features and their dependent components which include controllers, database_models, schema and routes.

## Type of change


## Trello Ticket ID

https://trello.com/c/KllQK32D

## How Can This Be Tested?

Pull this branch, run you server and head to apidocs by swagger. There shouln't appear any endpoint relating to the following groups:
1. Organisations
2. Organisation_admins
3. Organisation_members

## Checklist:
